src/events/client/ready.js

```diff
@@ -1,7 +1,9 @@
 /* eslint-disable no-await-in-loop */
-const reactionRoleConfig = require('../../../config/reactionRoleConfig.json')
+// import reactionRoleConfig from '../../../config/reactionRoleConfig.json'
+import { readFileSync } from 'fs'
+const reactionRoleConfig = JSON.parse(readFileSync('./config/reactionRoleConfig.json'))
```

Comment:
```suggestion
import reactionRoleConfig from '../../../config/reactionRoleConfig.json'
```
Utilisation d'import possible uniquement lors de l'utilisation du flag `--experimental-json-modules`. Pour l'instant, cette feature est à l'état expérimental (stability: 1) : https://nodejs.org/api/esm.html#esm_json_modules